client/components/follow-button/style.scss

```diff
@@ -1,3 +1,5 @@
+$follow-button-gray-disabled: lighten( $gray, 20% );
```

Comment:
Actually, sorry I forgot about the main button styles that it's inheriting. I think the repetition is fine. I would favor not creating new variables over it.